guides/common/assembly_preparing-environment-for-satellite-installation-ipv6.adoc

```diff
@@ -0,0 +1,9 @@
+[id="preparing-environment-for-satellite-installation-ipv6"]
+= Preparing your Environment for {Project} Installation in an IPv6 Network
+
+If you must provision hosts in IPv6 network, you can install {Project} in a IPv6 network and provision using UEFI HTTP Boot. Before installing {Project} in an IPv6 network, view the limitations that such scenario has and ensure that you meet the requirements.
```

Comment:
I'd use an impersonal form.
```suggestion
To provision hosts in IPv6 networks, {Project} should be installed in an IPv6 network and UEFI HTTP Boot configured. Before installing {Project} in an IPv6 network, view the limitations on the scenario and ensure that the requirements are met.
```

Reply:
I applied your technical corrections here and fixed the style a bit. The problem with an impersonal form, is that it is harder to translate and to understand. For example, with passive voice you do not see who performs the action. So as per the IBM style guide, we use active voice as often as possible.